### DISCUSSION TITLE
Disable Querying Without Fields Selected

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5847] [Feature Requests]
It's not very useful nor helpful that I can run a query without specifying any fields selected. 

Not a bug. It's just a UX Request that we prompt the user to at least select one field or by default use `Formatted` or `Aggregated`.

https://github.com/specify/specify7/assets/53784701/6b72d43d-1441-4035-86de-d8e73a1417e8

**Comment by maxpatiiuk** [comment]
It's very useful in development and might be useful for testing too - if you want to quickly get any record from a table, don't care which just so that you can open it in a form OR if you want to quickly run a query on a table, without carrying about specific fields or filters

is it that confusing to have this feature?

**Comment by emenslin** [comment]
possibly related to #3649
